dojo/api_v2/serializers.py

```diff
@@ -2938,7 +2938,7 @@ class Meta:
     def validate(self, data):
         async_updating = getattr(self.instance, 'async_updating', None)
         if async_updating:
-            for field in ['critical', 'high', 'medium', 'low']:
+            for field in ['critical', 'enforce_critical', 'high', 'enforce_high', 'medium', 'enforce_medium', 'low', 'enforce_low']:
                 old_days = getattr(self.instance, field, None)
                 new_days = data.get(field, None)
                 if old_days and new_days and (old_days != new_days):
```

Comment:
If the old value for one of the `enforce_` keys is `True` and the new value is `False` (or the reverse), this first condition will always evaluate to `False` and it wouldn't prevent modification, right?

Reply:
You are correct. Good catch -- these need to be `if old_days is not None and new_days is not None...`. Resolved in the latest commit.